.github/workflows/reusable-docker-workflow.yml

```
@@ -0,0 +1,55 @@
+name: Reusable Docker Workflow
+
+on:
+  workflow_call:
+    inputs:
+      # Target
+      repository:
+        required: true
+        type: string
+      version-ref:
+        required: true
+        type: string
+      
+      # Build
+      build-cmd:
+        required: true
+        type: string
+      
+      # Reference image
+      reference-image:
+        required: true
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
```

Add a name (which is used in the GitHub Actions UI)


```suggestion
  build:
    name: Reproducible build
    runs-on: ubuntu-24.04
```